### PR TITLE
Only show actual mutations from the cancer gene

### PR DESCRIPTION
**Pull request by aacic**
When the checkbox is selected in the Disco plot,  only show actual mutations from the cancer gene census and then increase to include other gene mutations when unchecked (max 10,000)

At the end of the Prioritize CGC label added text: add xx mutations out of yy total

When checking/unchecking CGC box, the number of mutations in legend is changed. 

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/plots/disco/data/DataMapper.ts [hunk 1]
```diff
@@ -67,23 +67,48 @@ export default class DataMapper {
...
 
-	constructor(settings: Settings, reference: Reference, sample: string, prioritizedGenes: Array<string> = []) {
+	private prioritizeGeneLabelsByGeneSets: boolean
+
+	constructor(
+		settings: Settings,
+		reference: Reference,
+		sample: string,
+		prioritizedGenes: Array<string> = [],
+		prioritizeGeneLabelsByGeneSets: boolean
```
Comment (siosonel) [inline client/plots/disco/data/DataMapper.ts:77]: @aacic shouldn't this configuration be tracked as `settings.prioritizeGeneLabelsByGeneSets`, and not as an independent argument?

Comment (aacic) [reply client/plots/disco/data/DataMapper.ts]: Good catch. It was already in settings.prioritizeGeneLabelsByGeneSets, I've just made a duplicate here. I've removed it. You can review it again.

**Review by xzhou82 — DISMISSED** [review]

File: client/plots/disco/label/Labels.ts [hunk 2]
```diff
@@ -9,10 +9,15 @@ export default class Labels extends Ring<Label> {
...
-	private filteredPrioritizedGenesList: Array<Label> = []
 	private overlapAngle: number
+	private prioritizeGeneLabelsByGeneSets: boolean
 
-	constructor(settings: Settings, elements: Array<Label>, hasPrioritizedGenes: boolean) {
+	constructor(
+		settings: Settings,
+		elements: Array<Label>,
+		hasPrioritizedGenes: boolean,
+		prioritizeGeneLabelsByGeneSets: boolean
```
Comment (siosonel) [inline client/plots/disco/label/Labels.ts:19]: @aacic there is also a duplicate argument here that could have used `settings.prioritizeGeneLabelsByGeneSets`?

Comment (aacic) [reply client/plots/disco/label/Labels.ts]: Yes. I've removed the duplicate.

**Review by siosonel — APPROVED** [review]

**Pull request merged**